src/vs/workbench/api/common/extHostEditorTabs.ts

```diff
@@ -48,16 +48,21 @@ export class ExtHostEditorTabs implements IExtHostEditorTabs {
 	}
 
 	$acceptEditorTabs(tabs: IEditorTabDto[]): void {
-		this._activeTab = undefined;
+		let activeIndex = -1;
+		let currentIndex = 0;
```

Comment:
*cough, cough* the index is being passed to the `map`-callback, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#parameters

Reply:
Fixed. I'm a purist what can I say, I like to implement everything by hand 😂😂